### PR TITLE
docs(Database): Clarify host value expected when running in docker

### PR DESCRIPTION
**Pull request by Carmageddon**
### SUMMARY

When running in Docker locally and attempting to connect to another DB also running in docker, the instructions are not clear enough, to help onboard new users, we should add a note clarifying what the HOST should be.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docs/docs/creating-charts-dashboards/creating-your-first-dashboard.mdx [hunk 1]
```diff
@@ -44,6 +44,8 @@ or for the purposes of this walkthrough, you can click the link below all these
 
 <img src={useBaseUrl("/img/tutorial/tutorial_03a_database_connection_string_link.png" )} width="600" />{" "} <br/><br/>
 
+Please note, if you are trying to connect to another locally running database such as Clickhouse (whether on host or another container), and you get the message `The port is closed.`, then you need to adjust the HOST to `host.docker.internal`
```
Comment (rusackas) [inline docs/docs/creating-charts-dashboards/creating-your-first-dashboard.mdx:47]: Is there something that's likely Clickhouse-specific here, or would it be best to remove the "such as Clickhouse" to keep it generic?

Comment (mistercrunch) [reply docs/docs/creating-charts-dashboards/creating-your-first-dashboard.mdx:47]: I think this part of the docs is meant for supporting UX more than admin/docker folks.

Feels like this note belongs somewhere between the "Connecting to Databases" and "Using Docker/Docker-compose" section

**Review by rusackas — APPROVED** [review]
No need to mention Clickhouse, but hey... it's an improvement! Merging :)

File: docs/docs/creating-charts-dashboards/creating-your-first-dashboard.mdx (hunk 1 above)
Comment (rusackas) [inline docs/docs/creating-charts-dashboards/creating-your-first-dashboard.mdx:47]:
```suggestion
Please note, if you are trying to connect to another locally running database (whether on host or another container), and you get the message `The port is closed.`, then you need to adjust the HOST to `host.docker.internal`
```

**Review by sfirke — APPROVED** [review]

**Pull request merged**